Extension/src/LanguageServer/extension.ts

```diff
@@ -404,6 +405,7 @@ export function registerCommands(enabled: boolean): void {
     commandDisposables.push(vscode.commands.registerCommand('C_Cpp.ExtractToMemberFunction', enabled ? () => onExtractToFunction(false, true) : onDisabledCommand));
     commandDisposables.push(vscode.commands.registerCommand('C_Cpp.ExpandSelection', enabled ? (r: Range) => onExpandSelection(r) : onDisabledCommand));
     commandDisposables.push(vscode.commands.registerCommand('C_Cpp.getIncludes', enabled ? (maxDepth: number) => getIncludes(maxDepth) : () => Promise.resolve()));
+    commandDisposables.push(vscode.commands.registerCommand('C_Cpp.OTFDocs', enabled ? onOTFDocs : onDisabledCommand));
```

Comment:
Same here, the command name string will technically be user visible so we should probably use something like `C_Cpp.ShowCopilotHover` instead. No need to change the rest of the code because users won't see that.

Reply:
Yeah, it shows on hover: 
![image](https://github.com/user-attachments/assets/e4001033-583a-4f20-977e-a6cb163d9491)